templates/plug_html_template.tmpl

```diff
@@ -0,0 +1,23 @@
+<html>
+	<body style="background-image:url('https://i.imgur.com/YWfFxsh.png');" link="white" vlink="white" alink="white">

```

Comment:
I'm not sure how I feel about a static image as the background. I think it would be neater if this was done with CSS. Also I'm not a fan of striped backgrounds.